frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerLifeCycle.java

```diff
@@ -45,6 +49,29 @@ public WorkerLifeCycle(ConfigManager configManager, Model model) {
         this.model = model;
     }
 
+    private String[] getEnvString(String cwd, String modelPath) {
+        ArrayList<String> envList = new ArrayList<>();
+        HashSet<String> blackList = configManager.getBlacklistEnvVars();
```

Comment:
I think use a single regex string for blackList might be better then delimited set